cmd/pingdom-exporter/main.go

```diff
@@ -33,6 +33,12 @@ var (
 		nil, nil,
 	)
 
+	pingdomMinRequestLimit = prometheus.NewDesc(
+		"pingdom_min_request_limit",
```

Comment:
Maybe we can name this something more explicit?

 ```suggestion
		"pingdom_rate_limit_min_remaining_requests",
```